src/modules/calculator/dtos/support.dto.ts

```diff
@@ -1,19 +1,18 @@
-import { IsEmail, IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { z } from 'zod';
 
-export class SupportDto {
-  @IsEmail()
-  @IsNotEmpty()
-  email: string;
+export const SupportDtoSchema = z.object({
+  email: z.string({
+    message: 'email must be a string and should not be empty',
+  }),
+  companyType: z.string({
+    message: 'companyType must be a string and should not be empty',
+  }),
+  employeesQuantity: z
+    .number({ message: 'wasteFootprint must be a number' })
+    .min(1, { message: 'wasteFootprint cannot be empty' }),
+  wasteFootPrint: z
+    .number({ message: 'wasteFootprint must be a number' })
+    .min(1, { message: 'wasteFootprint cannot be empty' }),
```

Comment:
Here too!

Reply:
You can use z.email() 


email: z.string().email(),  //only this is enough

email: z.string().email({message:"Email is required"}), // use this to show custom validation message